cinch/install-rhel7.yml

```diff
@@ -7,7 +7,7 @@
 # This playbook was tested with Ansible 1.8.4.
 
 - name: install cinch into a virtualenv on RHEL7
-  hosts: localhost
+  connection: local
```

Comment:
Fair point, I'd probably go for a combination:
- hosts: 127.0.0.1 //(or localhost)
  connection: local
as suggested in official Ansible docs.